api/v1alpha1/space_types.go

```diff
@@ -0,0 +1,80 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// SpaceCreatorLabelKey is used to label the Space with the ID of its creator (Dev Sandbox MasterUserRecord or AppStudio Workspace)
```

Comment:
in the case of DevSandbox, it will be more likely the UserSignup name - UserSignup controller will create the CR
```suggestion
	// SpaceCreatorLabelKey is used to label the Space with the ID of its creator (Dev Sandbox UserSignup or AppStudio Workspace)
```

Reply:
ok, thanks for the clarification